Modules/Sources/WordPressCore/Plugins/PluginService.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import UIKit
 import WordPressAPI
+@preconcurrency import WordPressAPIInternal
```

Comment:
We should not need to use the internal module. But we have to at the moment, because `PluginStatus` is not re-exported in the `WordPressAPI` module. I'll fix it properly in wordpress-rs later.